overrides/scripts/AdvRocketry.zs

```diff
@@ -413,7 +413,7 @@ alloy.recipeBuilder()
 
 // Suit Workstation
 recipes.remove(<advancedrocketry:suitworkstation>);
-makeShaped("ar_suit_workstation",
+makeShaped("ar_suit_workstation_legacy",

```

Comment:
Since you added a conversion recipe between the Workbench and the Crafting Station, I am not sure this legacy recipe is needed.

Reply:
I mostly made it just to make sure that this pattern would not break, though considering what it is making, I doubt anyone has a pattern for it, so its probably better to just switch directly to the crafting station recipe

Reply:
yeah, I would agree. This old recipe using the Forestry worktables should be removed, leaving just the one with the Crafting Station